pkg/services/grafana-apiserver/endpoints/request/request.go

```diff
@@ -24,3 +27,20 @@ func ParseOrgID(ns string) (int64, bool) {
 
 	return int64(orgID), true
 }
+
+type outputMediaType int
+
+const outputMediaKey outputMediaType = iota
```

Comment:
```suggestion
type outputMediaType struct{}
```

likely does not matter, but this is how we do context keys elsewhere

Reply:
👍 switched